src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -50,6 +51,21 @@ public static Name parseName(String name) throws ParseException {
         return new Name(trimmedName);
     }
 
+    /**
+     * Parses a {@code String name} into a {@code FolderName}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code folderName} is invalid.
+     */
+    public static FolderName parseFolderName(String folderName) throws ParseException {
+        requireNonNull(folderName);
+        String trimmedName = folderName.trim();
+        if (!Name.isValidName(trimmedName)) {
```

Comment:
I think you meant `FolderName.isValidName` instead of `Name.isValidName`? currently both methods are identical but in future the folder name vs name requirements might be different and we would want to use the appropriate `isValidName` methods

Reply:
My bad I overlooked this cos I used Name.isValidName at the start as a placeholder and forgot to change it to FolderName. Changing it now!